web/rainmaker/dev-packages/egov-wns-dev/src/ui-molecules-local/ActionFooter/index.js

```diff
@@ -60,9 +61,56 @@ class Footer extends React.Component {
           store.dispatch(setRoute(`/wns/apply?applicationNumber=${applicationNo}&connectionNumber=${connectionNumber}&tenantId=${tenantId}&action=edit&mode=MODIFY`));
        }
       };
+      const BillAmendment = {
+        label: "Edit",
+        labelKey: "WS_BILL_AMENDMENT_BUTTON",
+        link: async () => {
+
+          // checking for the due amount
+         
+          showHideAdhocPopup(this.props.state, store.dispatch, "connection-details");
+          // let due = getQueryArg(window.location.href, "due");
+          // let errLabel = (applicationNo && applicationNo.includes("WS"))?"WS_DUE_AMOUNT_SHOULD_BE_ZERO":"SW_DUE_AMOUNT_SHOULD_BE_ZERO";
+          // if(due && (parseInt(due) > 0)){            
+          //   toggleSnackbar(
+          //     true,
+          //     {
+          //       labelName: "Due Amount should be zero!",
+          //       labelKey: errLabel
+          //     },
+          //     "error"
+          //   );
+
+          //   return false;
+          // }
+
+          // check for the WF Exists
+          const queryObj = [
+            { key: "businessIds", value: applicationNos },
+            { key: "tenantId", value: tenantId }
+          ];        
+          
+          // let isApplicationApproved = await isWorkflowExists(queryObj);
+          // if(!isApplicationApproved){
+          //   toggleSnackbar(
+          //     true,
+          //     {
+          //       labelName: "WorkFlow already Initiated",
+          //       labelKey: "WS_WORKFLOW_ALREADY_INITIATED"
+          //     },
+          //     "error"
+          //   );
+          //   return false;
+          // }   
+          // store.dispatch(setRoute(`/wns/apply?applicationNumber=${applicationNo}&connectionNumber=${connectionNumber}&tenantId=${tenantId}&action=edit&mode=MODIFY`));
+       }
+      };
     //if(applicationType === "MODIFY"){
     downloadMenu && downloadMenu.push(editButton);
+    downloadMenu && downloadMenu.push(BillAmendment);
```

Comment:
add a condition to check whether we should show billamend button for that service , which is present in that search screen